src/pymor/models/basic.py

```diff
@@ -87,6 +87,44 @@ def __str__(self):
     def _compute_solution(self, mu=None, **kwargs):
         return self.operator.apply_inverse(self.rhs.as_range_array(mu), mu=mu)
 
+    @property
+    def dual(self):
+        if not hasattr(self, '_dual'):
```

Comment:
There's an idiom for this type of access that I'd consider more pythonic. 
```
try:
    return self._dual
except AttributeError:
    self._dual = self.with_(rhs=self.output_functional.H)
    return self._dual
```

Reply:
sounds good. thanks.